src/components/widgets/LabelStatic.vue

```diff
@@ -14,7 +14,7 @@ defineProps<{
         {{ label }}
       </span>
       <div :class="{ 'alert alert-warning': !modelValue }">
-        <span class="form-control">{{ modelValue }}</span>
+        <span class="form-control" style="border-top-left-radius: 0">{{ modelValue }}</span>
```

Comment:
I might prefer to avoid using an inline style here

Reply:
Buy it, is true that there is no inline styles in the code base so no reason to start using it.